src/NHSD.BuyingCatalogue.Ordering.Api/Services/CreatePurchasingDocument/CsvStreamStreamWriter.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace NHSD.BuyingCatalogue.Ordering.Api.Services.CreatePurchasingDocument
 {
-    internal sealed class CsvStreamStreamWriter<TEntity, TClassMap> : ICsvStreamWriter<TEntity>
+    public sealed class CsvStreamStreamWriter<TEntity, TClassMap> : ICsvStreamWriter<TEntity>
```

Comment:
Why change this? As above, shouldn't be necessary.

Reply:
I was getting error during implementaion but resolved with changes in `*.csproj `changes, so reverting back to `internal`